osu/map.py

```diff
@@ -52,9 +53,26 @@ def files(self):
 
     def zip(self):
         map_pack_file = MAP_PACKS_FOLDER.joinpath(f"{uuid.uuid4()}.zip")
-
+        print(map_pack_file)
         files = self.files()
         FileTools.zip_files(files, to_file=map_pack_file)
-        FileTools.delete_files(files)
 
         return map_pack_file
+
+    def upload(self, dbx_path: str) -> str | None:
+        zf = self.zip()
+        path = zf.name
+        dbx = dropbox.Dropbox(DROPBOX_ACCESS_TOKEN)
```

Comment:
this is a client that does not change between uploads

make a DropboxClient class with the client as a static variable (refer to osu client maybe)

btw if it isnt clear, client means a thing that receives stuff from a server which is why i called them clients

Reply:
I kinda get what you mean, but where would I initialise the client? Will currently initialise in this class only